mirar/processors/mask.py

```diff
@@ -19,6 +20,12 @@
 MASK_VALUE = np.nan
 
 
+class TooManyMaskedPixelsError(ProcessorError):
```

Comment:
You shouldn't put this error here if it does not get used in the processor

Reply:
I couldn't think of a good place to put this, I thought this went here because it was an error related to masking. It is only used in the winter specific files, idk if I want to put it there?

Reply:
Yeah, I think it would be better there since it's a WINTER-specific error.